File: tools/bench/answers/python_haiku/s1/08_invoice.py
items = [
    ("widget", 4, 2.5),
    ("gizmo", 2, 7.25),
    ("bolt", 10, 0.1)
]
total = 0
max_line_total = 0
max_item_name = ""
for name, qty, price in items:
    line_total = qty * price
    total += line_total
    if line_total > max_line_total:
        max_line_total = line_total
        max_item_name = name
print(total)
print(max_item_name)

File: tools/bench/answers/rust_haiku/s1/08_invoice.rs
fn main() {
    let items = [
        ("widget", 4, 2.5),
        ("gizmo", 2, 7.25),
        ("bolt", 10, 0.1),
    ];

    let mut total = 0.0;
    let mut max_item = "";
    let mut max_line_total = 0.0;

    for (name, qty, price) in items.iter() {
        let line_total = (*qty as f64) * price;
        total += line_total;

        if line_total > max_line_total {
            max_line_total = line_total;
            max_item = name;
        }
    }

    println!("{}", total);
    println!("{}", max_item);
}
